packages/devtools/lib/src/service_manager.dart

```diff
@@ -120,8 +120,7 @@ class ServiceConnectionManager {
     VmServiceWrapper service, {
     @required Future<void> onClosed,
   }) async {
-    const kServiceStreamOld = '_Service';
-    const kServiceStreamNew = 'Service';
+    final serviceStreamName = await service.serviceStreamName;
```

Comment:
@devoncarew FYI - I changed this slightly, since I ended up exposing the stream name elsewhere, so we don't have to handle both Service streams here now. LMK if you think there may be any consequences of changing this.